### DISCUSSION TITLE
I get an error: Enum value was out of legal range. (Parameter 'compressionLevel') in get_BrotliContent in a new project

### DISCUSSION THREAD

**Discussion by robban** [serenity-is/Serenity#6985] [Help]
Just trying to get started with the Serene template (On a mac using "dotnet new Serene.Templates"). I have started up the server and I can view the main page but I am getting an error message due to a 404 on the following request:

http://localhost:5000/DynJS.axd/Lookup.Administration.Language.js?v=1696925915038

This is the stack trace for the exception that occurs, any help would be appreciated.

> System.ArgumentException: Enum value was out of legal range. (Parameter 'compressionLevel')
>    at System.IO.Compression.BrotliUtils.GetQualityFromCompressionLevel(CompressionLevel compressionLevel)
>    at System.IO.Compression.BrotliStream..ctor(Stream stream, CompressionLevel compressionLevel)
>    at Serenity.Web.ScriptContent.get_BrotliContent() in /_/src/Serenity.Net.Web/DynamicScript/DynamicScript/ScriptContent.cs:line 105
>    at Serenity.Web.Middleware.DynamicScriptMiddleware.ReturnScript(HttpContext context, String scriptKey, String contentType, Boolean json) in /_/src/Serenity.Net.Web/DynamicScript/DynamicScript/DynamicScriptMiddleware.cs:line 125
>    at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
>    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
>    at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
> fail: Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware[1]
>       An unhandled exception has occurred while executing the request.
>       System.ArgumentException: Enum value was out of legal range. (Parameter 'compressionLevel')
>          at System.IO.Compression.BrotliUtils.GetQualityFromCompressionLevel(CompressionLevel compressionLevel)
>          at System.IO.Compression.BrotliStream..ctor(Stream stream, CompressionLevel compressionLevel)
>          at Serenity.Web.ScriptContent.get_BrotliContent() in /_/src/Serenity.Net.Web/DynamicScript/DynamicScript/ScriptContent.cs:line 105
>          at Serenity.Web.Middleware.DynamicScriptMiddleware.ReturnScript(HttpContext context, String scriptKey, String contentType, Boolean json) in /_/src/Serenity.Net.Web/DynamicScript/DynamicScript/DynamicScriptMiddleware.cs:line 125
>          at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
>          at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
>          at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
> Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware: Error: An unhandled exception has occurred while executing the request.
>

**Comment by volkanceylan** [comment]
You probably have .NET 7, not 6 installed which has a breaking change:

https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/7.0/brotlistream-ctor
